### ISSUE TITLE
Failure in getting SPAdes version

### ISSUE THREAD

**Issue by wanyuac** [sanger-pathogens/ariba#314]
ARIBA v2.14.6. The regular expression failed to extract the SPAdes version (v3.14.1).

```bash
$ ariba version
WARNING: I tried to get the version of spades with: "/home/yu.wan/.conda/envs/ariba/bin/spades.py --version" and the output didn't match this regular expression: "SPAdes\s+v([0-9\.]+)"
ARIBA version: 2.14.6

External dependencies:
bowtie2 2.3.5.1 /home/yu.wan/.conda/envs/ariba/bin/bowtie2
cdhit   4.8.1   /home/yu.wan/.conda/envs/ariba/bin/cd-hit-est
nucmer  3.1     /home/yu.wan/.conda/envs/ariba/bin/nucmer
spades  ERROR   /home/yu.wan/.conda/envs/ariba/bin/spades.py

External dependencies OK: True

Python version:
3.7.10 (default, Feb 26 2021, 18:47:35) 
[GCC 7.3.0]

Python packages:
ariba   2.14.6  /home/yu.wan/.conda/envs/ariba/lib/python3.7/site-packages/ariba/__init__.py
bs4     4.9.3   /home/yu.wan/.conda/envs/ariba/lib/python3.7/site-packages/bs4/__init__.py
dendropy        4.5.2   /home/yu.wan/.conda/envs/ariba/lib/python3.7/site-packages/dendropy/__init__.py
pyfastaq        3.17.0  /home/yu.wan/.conda/envs/ariba/lib/python3.7/site-packages/pyfastaq/__init__.py
pymummer        0.10.3  /home/yu.wan/.conda/envs/ariba/lib/python3.7/site-packages/pymummer/__init__.py
pysam   0.15.3  /home/yu.wan/.conda/envs/ariba/lib/python3.7/site-packages/pysam/__init__.py

Python packages OK: True

Everything looks OK: True

$ spades.py --version
SPAdes genome assembler v3.14.1
```

**Comment by rpetit3** [comment]
Hi @wanyuac! 

I submitted a PR that should fix this for you.

Cheers,
Robert

**Comment by puethe** [comment]
Resolved in #315
